metrics/tally/factory.go

```diff
@@ -48,14 +48,22 @@ func (f *factory) Gauge(options metrics.Options) metrics.Gauge {
 	return NewGauge(scope.Gauge(options.Name))
 }
 
-func (f *factory) Timer(options metrics.Options) metrics.Timer {
+func (f *factory) Timer(options metrics.TimerOptions) metrics.Timer {
 	scope := f.tally
 	if len(options.Tags) > 0 {
 		scope = scope.Tagged(options.Tags)
 	}
 	return NewTimer(scope.Timer(options.Name))
```

Comment:
hm, I thought tally supported buckets for timers. At least let's have a TODO, since we're dropping the buckets.

Reply:
No, Timer doesn't, but the tally.Histogram suports a DurationBucket - so depends if we want to switch underlying representation? Will mark as TODO for now.

Reply:
#67